tasks/gen/resource/browser.cr

```diff
@@ -134,12 +134,24 @@ class Gen::Resource::Browser < LuckyCli::Task
 
   private def columns_are_valid? : Bool
     column_definitions.any? && column_definitions.all? do |column_definition|
-      column_parts = column_definition.split(":")
-      column_name = column_parts.first
+      column_parts = parse_definition(column_definition)
+      column_name = column_parts.first?.to_s
```

Comment:
This is needed because you can do `"".split(":").first` and get nil but `([] of String).first` throws an exception 😕